nautobot/docs/additional-features/caching.md

```diff
@@ -189,3 +189,177 @@ CELERY_RESULT_BACKEND_TRANSPORT_OPTIONS = CELERY_BROKER_TRANSPORT_OPTIONS
 Please see the official Celery documentation for more information on how to [configure Celery to use Redis Sentinel](https://docs.celeryq.dev/en/stable/getting-started/backends-and-brokers/redis.html?highlight=sentinel#configuration).
 
 Please also see the [Nautobot documentation on required settings for Celery](../../configuration/required-settings#task-queuing-with-celery) for additional information.
+
+## Caching with TLS
+
+Redis can be configured to use TLS for additional security.
+
+### Without Sentinel
+
+#### `django-redis` TLS Configuration
+
+To enable TLS with `django-redis` task queuing, some additional options are required:
+
+```python
+import ssl
+
+CACHES["default"]["OPTIONS"]["CONNECTION_POOL_KWARGS"] = {
+    "ssl_cert_reqs": ssl.CERT_REQUIRED,
+    "ssl_ca_certs": "/opt/nautobot/redis/ca.crt",
+    "ssl_certfile": "/opt/nautobot/redis/tls.crt",
+    "ssl_keyfile": "/opt/nautobot/redis/tls.key",
+}
+```
+
+#### `django-cacheops` TLS Configuration
+
+To enable TLS with cacheops, the `CACHEOPS_REDIS` must be defined as a dictionary as follows:
+
+```python
+import ssl
+
+CACHEOPS_REDIS = {
+    "host": os.getenv("NAUTOBOT_REDIS_HOST", "localhost"),
+    "port": int(os.getenv("NAUTOBOT_REDIS_PORT", 6379)),
+    "password": os.getenv("NAUTOBOT_REDIS_PASSWORD", ""),
+    "ssl": True,
+    "ssl_cert_reqs": ssl.CERT_REQUIRED,
+    "ssl_ca_certs": "/opt/nautobot/redis/ca.crt",
+    "ssl_certfile": "/opt/nautobot/redis/tls.crt",
+    "ssl_keyfile": "/opt/nautobot/redis/tls.key",
```

Comment:
Just to make sure I'm reading this and below correctly - are all three of these config keys required here, or is it "pick one"?

Reply:
typically all 3 I think there are use cases for only 1 but every place I have seen it documented they list all 3.